cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -0,0 +1,61 @@
+{# ==========================================================================
+
+   expandable.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create an Expandable molecule when given:
+
+   label:                Label you want on the Expandable.
+                         Default is an empty string.
+
+   settings
+   settings.is_bordered: Whether the Expandable has a bottom border or not.
+                         Default is true.
+   settings.is_midtone:  Whether the Expandable is gray or not.
+                         Default is false.
+   settings.is_expanded: Whether the Expandable is expanded or not.
+                         Default is false.
+   ========================================================================== #}
+
+{% macro render(label="", settings) %}
+{# Add the __expanded modifier class by default so all expandables are open
+   when JS is switched off, but add a data-js-hook to tell JS to remove it. #}
+<div class="m-expandable
+            m-expandable__expanded
+            {{ 'm-expandable__borders' if settings.is_bordered else '' }}
+            {{ 'm-expandable__midtone' if settings.is_midtone else '' }}"
+     {{ 'data-js-hook=expanded' if settings.is_expanded else '' }}>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/2d3f92646844c768a3580edb6f54ddc548e7ac77